script/Ciel/Day 13/13_01_02_CIEL13_8.txt

```diff
@@ -126,12 +145,16 @@ All I want is to stay with you forever and find happiness, Senpai――no, Ciel.
 [sha:b9f14569516d3db4aa999126b6fbc41015a8d428]{
 -- Page 481, Offset 36645.
 -- 「なんで、もう先輩が何を言おうと騙されませんから。こうなったら地獄の底まで付いていって無理矢理笑わせます。
-"Well, no matter what you say, you won't be able to deceive me anymore. The way things are now, I'll follow you to the ends of hell and force a smile out of you.
+"That's why, no matter what you say, you won't be able to fool me anymore.
+ The way things are now, I'd go through hell and back just to make you smile.
```

Comment:
IT IS NOT INCORRECT

Reply:
Sorry bud but this crusade is misguided. Sticking an 'Anyway' on here deflates the mood quite a bit by making it sounds like he's changing the subject when he's not.

Reply:
It's not always incorrect, but it is in the vast majority of these cases. That, and the incidence is a ridiculous problem anyway.

Try hitting up "that's why" in normal novels; it is not common.

I tried LotR, and that's 500k words: 2 results
I tried a stephen king novel of about 100 pages: 4 results.
I tried an academic essay of about 300 pages: 3 results.

Meanwhile, the Tsukihime script currently has 284 or somewhere in that ballpark.

Reply:
Ok yeah that's a lot. I don't think that just slapping something else on the front starting in d13 is necessarily a solution though - if you want to avoid that's why, rewrite the whole phrase. I spot checked Grapes of Wrath, and it has 12 - 
- Start of sentence, raw: 6
- Start of sentence, and/so/well prefix: 5
- Mid-sentence: 1

so, a pretty even mix of prefixed and unprefixed, but yeah even for a fairly speech heavy novel not a lot. I'm fine with this specific example being an "_And_ that's why", "anyway" just feels like too much of a redirect.

I'd also be curious if Nasu's use of だから is in line with normal JP prose - is he just _writing more bad explanatory sentences_ because of the medium?